moveit_ros/planning_interface/test/serialize_msg.py

```diff
@@ -102,14 +102,14 @@ def test_rejectIntTuple(self):
 
     def test_rejectUnicode(self):
         with self.assertRaisesRegexp(Exception, "Python argument types in"):
-            self.helper.compareEmbeddedZeros(u"kdasd")
+            self.helper.compareEmbeddedZeros("kdasd")
```

Comment:
Can you keep unicode as in https://github.com/ros-planning/moveit/pull/3105/commits/02d7f0b6e5357c8e5f055ef7f4de6343cf91baf8